articles/native-platforms/android/01-login.md

```diff
@@ -3,24 +3,25 @@ title: Login
 description: This tutorial will show you how to integrate Lock 10 in your Android project in order to present a login screen.
 ---
 
-## Android - Login Tutorial
-
-This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your android apps with Auth0.
+This tutorial will show you how to integrate Lock 10 in your Android project in order to present a login screen.
 
 ::: panel-info System Requirements
 This tutorial and seed project have been tested with the following:
 
 * AndroidStudio 2.0
 * Emulator - Nexus5X - Android 6.0 
-  :::
+:::
 
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/inaka/auth0-android-sample/tree/01-login',
```

Comment:
@tenmaster Please update to the valid URL: https://github.com/auth0-samples/auth0-android-sample/tree/master/01-login